rotkehlchen/db/drivers/db_messages.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+
+enum DBConnectionType {
+  USER = 0;
+  TRANSIENT = 1;
+  GLOBAL = 2;
+}
+
+enum DBMethod {
+  INITIALIZE = 0;
+  OPEN_CURSOR = 1;
+  CLOSE_CURSOR = 2;
+  EXECUTE = 3;
+  EXECUTEMANY = 4;
+  EXECUTESCRIPT = 5;
+  ROLLBACK = 6;
+  COMMIT = 7;
+  CLOSE = 8;
+}
+
+enum DBError {
+  ArithmeticError = 0;
+  AssertionError = 1;
+  AttributeError = 2;
+  Exception = 3;
+  EOFError = 4;
+  FloatingPointError = 5;
+  GeneratorExit = 6;
+  ImportError = 7;
```

Comment:
I believe we shouldn't map 1:1 the python errors. Also lefteris https://discord.com/channels/657906918408585217/1238139584890535966/1279014760914092085 mentioned here. I would have something like:

- SystemError
- sqliteError
- ValueError

and maybe 1 or 2 more if you need to differenciate them. You can also add attributes if they are needed for debugging/processing like the name of the error